Description: Execute maintenance commands on PS printers instead of printing them
Bug: http://www.cups.org/str.php?L4007
Author: Till Kamppeter

--- a/cups/ppd-cache.c
+++ b/cups/ppd-cache.c
@@ -1272,7 +1272,7 @@
 
     if (filter)
       cupsArrayAdd(pc->filters,
-                   "application/vnd.cups-command application/postscript 0 -");
+                   "application/vnd.cups-command application/postscript 0 commandtops");
   }
 
   if ((ppd_attr = ppdFindAttr(ppd, "cupsPreFilter", NULL)) != NULL)
